bin/check_formatting.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:async';
+import 'package:exercism_dart/src/utils.dart';
+
+Future<Null> main() async {
+  final CommonUtils utils = new CommonUtils();
+
+  utils.fetchConfiglet().then((int exitCode) async {
+    if (exitCode == 0) {
+      print('Checking config.json formatting...');
+      await utils.runCmd('bin/configlet', ['lint', '--track-id', 'dart', '.']);
+
+      print('Checking all Dart files formatting...');
+      await utils.runCmd('pub', ['run', 'dart_style:format', '-l', '120', '-n', '.']);
```

Comment:
Only perform configlet related commands inside the `fetchConfiglet().then` callback.